python/CCLE_data.py

```diff
@@ -219,7 +325,7 @@ def Heatmap(data, size, colour, title):
 REORGANIZING DATA
 """
```

Comment:
Okay, this looks like you're running all the functions. This can be in a separate script called 'exploratory_correlation_analysis.py` or something to that effect.